src/eos-autoupdater.c

```diff
@@ -1,5 +1,9 @@
-#include "eos-updater-generated.h"
+/* -*- mode: C; c-file-style: "gnu"; indent-tabs-mode: nil; -*-
+ * TODO: Copyright?
```

Comment:
Yeah, I added that TODO, because I'm not sure what copyright it should be and I haven't investigated what it should be. It is basically a drive by comment. ;)